jupyter-book/preamble.md

```diff
@@ -12,53 +12,57 @@ Interested in contributing? Please have a look at https://github.com/theislab/si
+- **AnnData and Scanpy**: While prior experience with these tools is helpful, it is not strictly required. This book provides a detailed introduction to AnnData and outlines the workflow for working with Scanpy. However, it does not cover the full range of Scanpy’s functionalities. To deepen your understanding, we recommend exploring the [scanpy tutorials](https://scanpy.readthedocs.io/en/stable/tutorials.html) and referring to the [scanpy API reference](https://scanpy.readthedocs.io/en/stable/api.html) as needed.
 
-- If you are interested in multimodal data analysis, the basics of muon and MuData are recommended. This book covers MuData in greater detail, but only briefly introduces muon analogously to AnnData and scanpy. The excellent [muon tutorials](https://muon-tutorials.readthedocs.io/en/latest/) serve as a great introduction to multimodal data analysis with muon.
+- **Multimodal Data Analysis**: For readers who have interest in multimodal data analysis, understanding tools like muon and MuData is beneficial. The [muon tutorials](https://muon-tutorials.readthedocs.io/en/latest/) provide a solid introduction to this field.
 
-- Basic R programming. Familiarity with control flow and basic data structures suffices. If you are new to programming and R we recommend the free [R for data science](https://r4ds.had.co.nz/) book.
+- **Basic R Programming**: Knowledge of control flow and basic data structures suffices. New learners can refer to [R for data science](https://r4ds.had.co.nz/) for a comprehensive introduction.
 
-- Basics of biology. While we roughly introduce the generation of the data, we will not cover the fundamentals of {term}`DNA`, RNA and proteins. If you are completely new to molecular biology in general, it might be advisable to work through _Molecular Biology of the Cell_ by Bruce Alberts et al.
+- **Basic Biology**: While this book offers a rough overview of data generation, it does not cover fundamental topics like {term}`DNA`, RNA, and proteins. _Molecular Biology of the Cell_ by Bruce Alberts et al. is a recommended resource for those new to molecular biology.
 
 ## Peer-review
 
-Although most of our chapters have been reviewed by multiple authors and the editors of this book and external experts, this book has not been officially peer-reviewed. We therefore strongly encourage readers to review our chapters and to kindly provide constructive feedback. We are more than happy to adapt, improve or overhaul the content if necessary.
+Although the content has been reviewed by multiple authors, editors, and external experts, this book has not undergone formal peer review. We encourage readers to provide constructive feedback to help refine and improve the material.
```

Comment:
```suggestion
Although the content has been reviewed by multiple authors, editors, and external experts, this book has not undergone formal peer review.
We encourage readers to provide constructive feedback to help refine and improve the material.
```

Can you add some details here how to do it? Stuff like where to open an issue, adding reasoning and sources etc blablabal